recipes/paho-mqtt/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "paho-mqtt" %}
+{% set version = "1.5.0" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  fn: '{{ name }}-{{ version }}.tar.gz'
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: e3d286198baaea195c8b3bc221941d25a3ab0e1507fc1779bdb7473806394be4
+
+build:
+  number: 0
+  noarch: python
+  script: {{ PYTHON }} setup.py install --old-and-unmanageable
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
+
+test:
+  imports:
+    - paho
+    - paho.mqtt
+  requires:
+    - mock
+    - pylama
+    - pytest
+    - six
+
+about:
+  home: http://eclipse.org/paho
+  license: EPL (Eclipse) v1.0 / EDP (Eclipse Distribution L...) v1.0
+  license_family: OTHER
+  license_file: LICENSE.txt
+  summary: MQTT version 3.1.1 client class
+  description: Eclipse Paho MQTT Python Client
+
+extra:
+  recipe-maintainers:
+    - ilanschnell
+    - nerohmot
```

Comment:
@nerohmot are you willing to be a maintainer?

Reply:
yes